vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt

```diff
@@ -283,6 +283,7 @@ class UiAllScreensSanityTest {
         clickListItem(R.id.matrixProfileRecyclerView, 9)
         // File tab
         clickOn(R.string.uploads_files_title)
+        sleep(1000)
```

Comment:
whilst running the tests locally I was seeing some flaky behaviour around closing dialogs and switching viewpagers tabs, the sleeps aren't ideally but the tests run too fast without them!

Reply:
OK, this is not the first occurrence (I think) in this test. I would like to find some active triggering to continue the test instead of waiting, but I have now idea how to do that today.
Adding too many sleep() will slow down the entire test suite execution, and we want tests to be fast. But of course you already know that :)

Reply:
But anyway I prefer the test to pass at first shot instead of trying again because of some flaky behavior :)